Externals/wxWidgets3/include/wx/bitmap.h

```diff
@@ -305,13 +302,7 @@ wxBitmap::
 #endif
 ConvertToDisabled(unsigned char brightness) const
 {
-    // XXX comex: scale support
-    wxImage disabledImage = ConvertToImage().ConvertToDisabled(brightness);
-    #ifdef __APPLE__
-    return wxBitmap(disabledImage, -1, GetScaleFactor());
-    #else
-    return disabledImage;
-    #endif
+    return ConvertToImage().ConvertToDisabled(brightness);
```

Comment:
We have local changes to wxw which can't be expected to be pushed upstream. We need to make sure that stuff is still applied...if applicable...